frontend/compositions/search/search-panel.tsx

```diff
@@ -58,10 +59,24 @@ export const SearchPanel = () => {
           {incidentSearch ? (
             <div>
               <div>
-                <PrimaryInput inputName={LOCATION} />
+                <PrimaryInput
+                  inputName={LOCATION}
+                  tooltipProps={{
+                    type: TooltipTypes.USEFORCE,
+                    icon: TooltipIcons.INFO,
+                    iconSize: "xs"
```

Comment:
Wondering if these have to be props? It seems only the type changes?

Could we map to these based on the input name within the PrimaryInput component?

Reply:
i didn't actually write this -- I'll have to leave it to the search-panel/table person to move this forward.